packages/flp-config-inquirer/src/translations/flp-config-inquirer.i18n.json

```
@@ -1,7 +1,7 @@
 {
     "prompts": {
         "inboundIds": "Inbound ID",
-        "emptyInboundsInfo": "Original application target mapping and the default parameters can be found in Fiori reference library under Implementation Information -> Configuration -> Target Mapping(s) in",
+        "emptyInboundsInfo": "The original application target mapping and the default parameters are found in the SAP Fiori reference library under Implementation Information -> Configuration -> Target Mappings in",
```

Is Target Mappings in correct? Should it be in something or is that concatenated?

---

I checked, and in the SAP Fiori Apps Reference Library, they provide a table with a list under Target Mappings, so I'd recommend to remove 'in'
```suggestion
        "emptyInboundsInfo": "The original application target mapping and the default parameters are found in the SAP Fiori reference library under Implementation Information -> Configuration -> Target Mappings",
```